cdap-ui/app/directives/log-viewer/log-viewer.less

```diff
@@ -14,30 +14,282 @@
+      height: 50px;
+      width: 100%;
+      min-width: 940px;
+
+      div {
+        display: inline-block;
+      }
+
+      .log-stats-container {
+        width: 35%;
+        min-width: 150px;
+        height: 50px;
+        float: left;
+        color: white;
+        background-color: @headerbar-background-color;
+        display: table;
+        margin: auto;
+
+        .log-stats {
+          height: 50px;
+          min-width: 40px;
+          display: table-cell;
+          vertical-align: middle;
+          text-align: center;
+          font-weight: bold;
+          top: -5px;
+
+          .stats-wrapper {
+            position: relative;
+          }
+
+        }
+        .log-stats-total {
+          width: 140px;
+          min-width: 140px;
+        }
+        .log-stats-errors {
+          width: 90px;
+          min-width: 90px;
+        }
+        .log-stats-warnings {
+          width: 115px;
+          min-width: 115px;
+        }
+        .stat-name {
+          color: @headerbar-text-color;
+          margin-left: 10px;
+          margin-right: 4px;
+        }
+        .stat-count {
+          width: 30px;
+          height: 15px;
+          border-radius: 8px;
+          color: black;
+          display: inline-block;
+          position: relative;
+          top: 2px;
+        }
+        .stat-count-number {
+          position: relative;
+          top: -2px;
+        }
+        .total-num {
+          background-color: #dfdfdf;
+        }
+        .errors-num {
+          background-color: red;
+        }
+        .warnings-num {
+          background-color: orange;
+        }
+      }
+
+      .log-actions {
+        width: 27%;
+        min-width: 410px;
+        height: 50px;
+        float: right;
+        background-color: @headerbar-background-color;
+        display: table;
+        margin: auto;
+
+        .log-action-container {
+          height: 50px;
+          display: table-cell;
+          vertical-align: middle;
+        }
+        .log-action-btn {
+          height: 25px;
+          width: 100px;
+          border: solid 1px white;
+          color: white;
+          background-color: @headerbar-background-color;
+          border-radius: 4px;
+          margin-right: 3px;
+          outline: grey;
+        }
+        .search {
+          position: relative;
+        }
+        .search .fa-search {
+          position: absolute;
+          top: 7px;
+          right: 8px;
+        }
+
+        .log-searchbar {
+          height: 25px;
+          width: 140px;
+          padding-right: 20px;
+          border-radius: 5px;
+          outline: none;
+          color: @searchbar-input-color !important;
+          font-weight: bold;
+          text-indent: 5px;
+        }
+        .log-searchbar:focus {
+          color: @searchbar-input-color !important;
+        }
+        .log-searchbar::-webkit-input-placeholder {
+          color: @searchbar-placeholder-color !important;
```

Comment:
also verify with @bdmogal and Sean, but I don't think we need to worry about placeholder color